### DISCUSSION TITLE
Changing @reference to \cite in LaTeX/PDF outputs

### DISCUSSION THREAD

**Discussion by jcarpi** [quarto-dev/quarto-cli#4412] [Q&A]
For PDF outputs, citing a reference in a document body results in `\textcite` in the LaTeX output. Is it possible to change the default behavior to `\cite`?

**Comment by dragonstyle** [comment]
I haven't found a way to get that to happen. When the `cite-method` is `biblatex`, the various syntaxes for authors result in:

```
@entry1       \textcite{entry1}
[@entry1]     \autocite{entry1}
[-@entry1]    \autocite*{entry1}
```

You write a filter that targets cites and emits the specific latex that you'd like, if you'd like to give that a try... Other's may know of a way to do this without the use of a filter.

**Reply by jcarpi** [reply]
Good info, thanks! I do have an easy workaround:

```
include-in-header:
  text: |
    \global\let\textcite\cite
```

**Reply by dragonstyle** [reply]
That is nice! Thx for sharing...
